community/kernel/src/main/java/org/neo4j/kernel/builtinprocs/BuiltInProcedures.java

```diff
@@ -185,4 +203,24 @@ private ProcedureResult( ProcedureSignature signature )
             this.signature = signature.toString();
         }
     }
+
+    //When we have decided on what to call different indexes
+    //this should probably be moved to some more central place
+    private enum IndexType
+    {
+        NODE_LABEL_PROPERTY( "node_label_property" ),
+        NODE_UNIQUE_PROPERTY( "node_unique_property" );
```

Comment:
I was under the impression that they were named "Property Uniqueness Constraint" and "Property Existence Constraint"?